springboot3-todo/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,30 @@
+####
+# This Dockerfile is used in order to build a container that runs the Quarkus application in JVM mode
+#
+# Before building the docker image run:
+#
+# mvn package
+#
+# Then, build the image with:
+#
+# docker build -f src/main/docker/Dockerfile -t spring/hello .
+#
+# Then run the container using:
+#
+# docker run -i --rm -p 8080:8080 --cpus=1 --memory=1G --name=spring spring/hello
+#
+###
+FROM fabric8/java-alpine-openjdk17-jre
```

Comment:
The container image build was failing for me, I can not see this image of docker or quay.io, does it exist?

Reply:
No idea. I copied this `Dockerfile` from the other project & just changed `openjdk11` to `openjdk17`. This `Dockerfile` (and the one in the `spring-todo` look really old.

Reply:
Ok, openjdk17 does not exist, we can probably drop this docker file as we are not using it for any tests atm

Reply:
If these Dockerfiles are being used then they should be updated, specifically because we will update to java17, in which case a java 17 base image will need to be used. Not sure if that can be supplied as a variable by the job?

If I changed it here it will break the job until the job is updated accordingly.

Reply:
I will open an issue to update the docker files, they are not being used in any jobs afaics atm